### ISSUE TITLE
Javascript is not ES5, yet is not transpiled down

### ISSUE THREAD

**Issue by OvermindDL1** [grych/drab#22]
Currently trying to run version 0.5.0 in my host of tests at work and I'm getting the error of:
```
Object doesn't support property or method 'forEach'
```
On the generated from Drab javascript code of:
```javascript
Drab.disable_drab_objects = function(disable) {
  
    document.querySelectorAll("[drab-event]").forEach(function(element) {
      element['disabled'] = disable
    })
  
}
```
Of which obviously the return of `document.querySelectorAll` does not have `forEach` method in ES5, and as this javascript is not being transpiled down, then it is not working.  This needs to use a for loop, like as such:
```javascript
Drab.disable_drab_objects = function(disable) {
  
    var arr = document.querySelectorAll("[drab-event]");
    for(var i=0; i<items.length; i++) {
      var element = item[i];
      element['disabled'] = disable
    }
  
}
```
And there are probably other areas that are using functions that do not exist in ES5.  Might be good to run the entire drab output through a transpiler like babel and see what it changes to fix that in the outputted javascript.

Right now I'm getting javascript errors in browsers that I still have to support.  ^.^;

**Comment by grych** [comment]
Thanks!
I've transpiled all JS templates with babel, there were few changes, but not very significant. About forEach, despite it is quite handy, you are right: I should focus on better compatibility than handy methods.

So I replaced every occurance of forEach with the for loop. Will go live in 0.5.1

**Comment by OvermindDL1** [comment]
Whoo!  I look forward!  :-D

Thanks much!

And yeah, would not have been a lot of changes, but they needed to be done.  ^.^

**Comment by grych** [comment]
Fixed in v0.5.1
